### PR TITLE
Update app icons and 'Your photos' icon

### PR DESCRIPTION
**Pull request by jancborchardt**
This should fix the issue with the wrong favicon, and it also compacts the "Your photos" icon.

What I couldn’t figure out is how to make the "Your photos" navigation entry show the correct icon. Simply changing the `icon-photos` class to `icon-yourphotos` displays nothing, but I don’t get where the icon is loaded from.

Please advise @nextcloud/photos

### CODE REVIEW

File: img/app.svg [hunk 1]
```diff
@@ -1 +1,4 @@
-<svg xmlns="http://www.w3.org/2000/svg" height="32" width="32" version="1"><path d="M2.8 4a1.3 1.3 0 00-1.3 1.3v22.4c0 .6.7 1.3 1.3 1.3h26.4c.6 0 1.3-.7 1.3-1.3V5.3c0-.6-.7-1.3-1.3-1.3zm.7 2h25v19h-25z" fill="#fff"/><circle cx="8.5" cy="11.2" r="3" fill="#fff"/><path d="M26.4 14.5l-4.7 6.2L20 23l-1.6-1.8-4.5-4.6-6 5.7-4.7 4.3h26.2v-8.7z" fill="#fff"/></svg>
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (skjnldsv) [inline img/app.svg]: Oh, it could be because of the missing xml?

Comment (skjnldsv) [reply img/app.svg]: I indeed changed something after passing it to scour!

File: img/app.svg [hunk 2]
```diff
@@ -1 +1,4 @@
...
+<svg xmlns="http://www.w3.org/2000/svg" height="32" width="32" version="1.0" viewBox="0 0 32 32">
+ <path style="color:#000000;block-progression:tb;text-transform:none;text-indent:0" fill="#fff" d="m2.6876 4c-0.395 0.0765-0.6946 0.4668-0.6875 0.875v22.25c0 0.46 0.424 0.876 0.875 0.876h26.25c0.4511-0.000012 0.87498-0.41692 0.875-0.875v-21.906c-0.001-0.6731-0.529-1.2229-1.031-1.219zm1.3124 1.9999 24 0.000014v10l-2-2-6 8-6-6-8 8h-2zm5.0002 2c-1.6569 0-3 1.3431-3 3s1.3431 3 3 3 3-1.3431 3-3-1.3431-3-3-3z"/>
```
Comment (skjnldsv) [inline img/app.svg]: the style is most likely not needed ;)

File: img/app.svg (hunk 1 above)
Comment (jancborchardt) [reply img/app.svg]: There you go :DD

File: img/app.svg (hunk 2 above)
Comment (jancborchardt) [reply img/app.svg]: Good point, fixed.

**Comment by jancborchardt** [comment]
@skjnldsv ready from my side. What is needed to show the `yourphotos.svg` as the navigation? Or should it be merged as is – then only the single photo icon will be shown as nav icon for "Your photos" though.

**Comment by skjnldsv** [comment]
> What is needed to show the `yourphotos.svg` as the navigation?

https://github.com/nextcloud/photos/blob/master/css/icons.scss#L27
https://github.com/nextcloud/photos/blob/master/src/Photos.vue#L29

**Comment by jakobroehrl** [comment]
Sorry, I couldn't see any changes on the icons. 
What exactly should change?

**Comment by skjnldsv** [comment]
> What exactly should change?

the favicon ;)

**Comment by jancborchardt** [comment]
> > What is needed to show the `yourphotos.svg` as the navigation?
> 
> https://github.com/nextcloud/photos/blob/master/css/icons.scss#L27
> https://github.com/nextcloud/photos/blob/master/src/Photos.vue#L29

Jeez, thanks a lot – not sure why I didn’t look into the CSS folder …

-----------

All fixed now:
nav icons

How does the favicon look for you? Should be the single image icon, in white, on blue. On all my browsers it’s still the old one (double image in black on blue) and I can’t clear it somehow.

**Comment by skjnldsv** [comment]
> Jeez, thanks a lot – not sure why I didn’t look into the CSS folder …

this is a bit counter intuitive, I only use it for the svg colour api ;)

**Comment by skjnldsv** [comment]
I think this is broken for me, but not from here, lots of my apps doesn't have the favicon :(
If it works for you, let's merge! :muscle:

**Comment by jancborchardt** [comment]
My cache has issues. @jakobroehrl can you test, and show a screenshot of how the favicon looks for you?

**Comment by jakobroehrl** [comment]
> My cache has issues. @jakobroehrl can you test, and show a screenshot of how the favicon looks for you?

Cleared the cache, reopend the browser but I only see the default nextcloud favicon from the files App.

**Comment by skjnldsv** [comment]
Same :thinking:

**Comment by jancborchardt** [comment]
Rebased to master – let’s get it merged. :)

We can still fix the favicon issue merged. If it simply shows the Files favicon it’s not so bad.

**Comment by jancborchardt** [comment]
(Maybe @juliushaertl can help next week when he’s back from vacation.)

**Comment by skjnldsv** [comment]
> If it simply shows the Files favicon it’s not so bad.

it's not even the files icon, it's the nc logo :p

**Comment by jancborchardt** [comment]
> > If it simply shows the Files favicon it’s not so bad.
> 
> it's not even the files icon, it's the nc logo :p

That’s fine too :D

**Comment by jancborchardt** [comment]
@skjnldsv merged branches can be deleted :) It's set up to do it automatically

**Comment by skjnldsv** [comment]
Oups :p
Old habits :see_no_evil: ;)

**Pull request merged**